collect_app/src/main/AndroidManifest.xml

```diff
@@ -174,11 +174,11 @@ the specific language governing permissions and limitations under the License.
          -->
 
         <provider
-            android:name=".provider.FormsProvider"
+            android:name=".external.FormsProvider"
```

Comment:
Are there any implications for apps that currently use these providers?

Reply:
It shouldn't, as you can only access content providers via a URI as far as I'm aware.